release-notes.txt

```diff
@@ -1,3 +1,511 @@
+RIOT-2024.10 "FlexTape" - Release Notes
+============================
+RIOT is a multi-threading operating system which enables soft real-time
+capabilities and comes with support for a range of devices that are typically
+found in the Internet of Things: 8-bit and 16-bit microcontrollers as well as
+light-weight 32-bit processors.
+
+RIOT is based on the following design principles: energy-efficiency, soft
+real-time capabilities, small memory footprint, modularity, and uniform API
+access, independent of the underlying hardware (with partial POSIX compliance).
+
+RIOT is developed by an international open-source community which is
+independent of specific vendors (e.g. similarly to the Linux community) and is
+licensed with a non-viral copyleft license (LGPLv2.1), which allows indirect
+business models around the free open-source software platform provided by RIOT.
+
+
+About this release
+==================
+
+Highlights of the 2024.10 release:
+
+This release fixes some long-standing packet buffer leaks that occurred when using
+`gnrc_netif_pktq` and 6LoWPAN or `netdev_new_api` (#20983, #20834).
+
+A long standing-bug in the CC2538 radio driver has been fixed when processing 
+corrupted frames, that before could render the device unresponsive. This can be 
+triggered due to radio interference, but we also believe that an attacker within range 
+of the radio could send crafted frames to reliably trigger the issue. We therefore 
+consider this a denial of service vulnerability and advise all users of the CC2538 to 
+upgrade to 2024.10 or cherry-pick 1b61216da734fcd40ae0d832b800e4e67975bdc8. More 
+details are published in the [security advisory](https://github.com/RIOT-OS/RIOT/security/advisories/GHSA-m75q-8vj8-wppw).
```

Comment:
```suggestion
A long standing-bug in the CC2538 radio driver has been fixed when processing
corrupted frames, that before could render the device unresponsive. This can be
triggered due to radio interference, but we also believe that an attacker within range
of the radio could send crafted frames to reliably trigger the issue. We therefore
consider this a denial of service vulnerability and advise all users of the CC2538 to
upgrade to 2024.10 or cherry-pick 1b61216da734fcd40ae0d832b800e4e67975bdc8. More
details are published in the [security advisory](https://github.com/RIOT-OS/RIOT/security/advisories/GHSA-m75q-8vj8-wppw).
```